static/spec/openapi.yaml

```diff
@@ -136,9 +188,14 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ExportStatus'
+        '500':
+          description: Error while encoding
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
```

Comment:
I think this one's description should be a bit more generic.

Reply:
Changed!